java/Nikiforov-Oleksandr/src/com/zoolatech/lecture4/tasks/_7/Task7.java

```diff
@@ -0,0 +1,60 @@
+package com.zoolatech.lecture4.tasks._7;
+
+import java.util.Scanner;
+
+/**
+ * Create a method that accepts a string of digits and after each digit appends
+ * its string representation in parentheses.
+ */
+
+public class Task7 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String digits = sc.next();
+        outputDigits(digits);
+    }
+
+    public static void outputDigits(String digits) {
+        char[] charArray = digits.toCharArray();
+        StringBuilder result = new StringBuilder();
+        for (char c : charArray) {
+            switch (c) {
+                case '1' -> {
+                    result.append(c).append("(one)");
```

Comment:
Some improvements that you can apply to your code:
1. `result.append(c)` is done in all case blocks - I think it can be extracted into a single place
2. the string representation is also appended in all blocks - the code duplication can be removed with a help of, for example, switch expression

Reply:
All issues fixed, can you check homework#2 https://github.com/zoolacode/zoola-academy-homework/pull/97